frontend/src/routes/(authenticated)/project/[project_code]/AddProjectMember.svelte

```diff
@@ -40,7 +40,6 @@
     type="email"
     label={$t('admin_dashboard.column_email')}
     bind:value={$form.email}
-    required
```

Comment:
is there a reason we don't want to use the required attribute?

Reply:
It adds browser validation that
1) Is redundant
2) Doesn't include our custom validation message

![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/fef2fdd9-efa7-4677-8f31-16cf8698043a)